scapy/packet.py

```diff
@@ -223,6 +223,26 @@ def __init__(self,
         Optional[bytes],
     ]
 
+    @property
+    def comment(self):
+        # type: () -> Optional[bytes]
+        """Get the comment of the packet"""
+        if self.comments is not None and len(self.comments) > 0:
+            return self.comments[-1]
+        return None
+
+    @comment.setter
+    def comment(self, value):
+        # type: (Optional[bytes]) -> None
+        """
+        Set the comment of the packet.
+        If value is None, it will clear the comments.
+        """
+        if value is not None:
+            self.comments = [value]
+        else:
+            self.comments = None
```

Comment:
```suggestion
        self.comments = value
```

Reply:
Got it.

As self.comment is None by default, the following should be enough:
if value is not None:
            self.comments = [value]